scripts/report_errors.sh

```diff
@@ -44,6 +43,26 @@ function post_error_to_github
       --header 'content-type: application/json' \
       -u stackenbotten:$GITHUB_ACCESS_TOKEN \
       --data "{\"body\": \"There was an error when running \`$CIRCLE_JOB\` for commit \`$CIRCLE_SHA1\`:\n\`\`\`\n$FORMATTED_ERROR_MSG\n\`\`\`\nPlease check that your changes are working as intended.\"}"
+
+    post_review_comment_to_github
+}
+
+function post_review_comment_to_github {
+    GITHUB_API_URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$CIRCLE_PR_NUMBER/comments"
+
+    sed -i 1d $ERROR_LOG
+
+    while read line; do
+        TESTING_PATH=$(echo $line | grep -oE ".*\.cpp")
```

Comment:
Rather name this `ERROR_PATH` and `ERROR_LINE`?

Reply:
Ah yes, leftover from earlier changes, good point